### ISSUE TITLE
cosmos:browserify error in project using Meteor + React

### ISSUE THREAD

**Issue by NoahGrumman** [elidoran/cosmos-browserify#43]
I'm building a web application using Meteor + React and the project suddenly no longer runs and browserify gives the incomprehensible error seen below.

A friend cloned the same repository and it run perfectly on his computer. I suspect that it has to do with some sort of incompatibility between specific meteor packages and npm packages but have no idea of how to go about fixing it. I've tried deleting and reinstalling meteor, uninstalling and reinstalling cosmos:browserify, and even downloading the entire meteor project anew. No success.

Note that "/Users/noahgrumman/.meteor/packages/meteor-tool/..." is not the project directory but instead the directory where meteor packages are downloaded.

Any ideas about what could be going wrong would be greatly appreciated.

```
=> Started proxy.
=> Started MongoDB.
=> Errors prevented startup:

   While processing files with cosmos:browserify (for target web.browser):
   native: Converting circular structure to JSON
   at Object.stringify (native)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/isopackets/ddp/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/templating/.1.1.5.7x2gis++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/coffeescript/.1.0.11.iqzsj2++os+web.browser+web.cordova/plugin.compileCoffeescript.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/less/.2.5.1.196vlrt++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at BrowserifyPlugin.buildErrorMessage (<runJavaScript-61>:177:70)
   at BrowserifyPlugin.compileOneFile (<runJavaScript-61>:142:42)
   at <runJavaScript-59>:555:47
   at <runJavaScript-59>:585:11
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:328:15
   at Object.async.forEachLimit.async.eachLimit
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:221:35)
   at BrowserifyPlugin.processFilesForTarget (<runJavaScript-59>:530:11)


   While processing files with cosmos:browserify (for target os.osx.x86_64):
   native: Converting circular structure to JSON
   at Object.stringify (native)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/isopackets/ddp/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/templating/.1.1.5.7x2gis++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/coffeescript/.1.0.11.iqzsj2++os+web.browser+web.cordova/plugin.compileCoffeescript.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/less/.2.5.1.196vlrt++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at BrowserifyPlugin.buildErrorMessage (<runJavaScript-61>:177:70)
   at BrowserifyPlugin.compileOneFile (<runJavaScript-61>:142:42)
   at <runJavaScript-59>:555:47
   at <runJavaScript-59>:585:11
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:328:15
   at Object.async.forEachLimit.async.eachLimit
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.pipsf++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:221:35)
   at BrowserifyPlugin.processFilesForTarget (<runJavaScript-59>:530:11)


=> Your application has errors. Waiting for file change.
^C
Noahs-MacBook-Pro-2:Conch noahgrumman$ meteor
noahgrumman, this is your first time using Meteor!
Installing a Meteor distribution in your home directory.
######################################################################## 100.0%
[[[[[ ~/Desktop/Conch ]]]]]

=> Started proxy.
=> Started MongoDB.
=> Errors prevented startup:

   While processing files with cosmos:browserify (for target web.browser):
   native: Converting circular structure to JSON
   at Object.stringify (native)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/isopackets/ddp/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/templating/.1.1.5.7x2gis++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/coffeescript/.1.0.11.iqzsj2++os+web.browser+web.cordova/plugin.compileCoffeescript.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/less/.2.5.1.196vlrt++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at BrowserifyPlugin.buildErrorMessage (<runJavaScript-61>:177:70)
   at BrowserifyPlugin.compileOneFile (<runJavaScript-61>:142:42)
   at <runJavaScript-59>:555:47
   at <runJavaScript-59>:585:11
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:328:15
   at Object.async.forEachLimit.async.eachLimit
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:221:35)
   at BrowserifyPlugin.processFilesForTarget (<runJavaScript-59>:530:11)


   While processing files with cosmos:browserify (for target os.osx.x86_64):
   native: Converting circular structure to JSON
   at Object.stringify (native)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/meteor-tool/.1.1.10.1b51q9m++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/isopackets/ddp/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/templating/.1.1.5.7x2gis++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/coffeescript/.1.0.11.iqzsj2++os+web.browser+web.cordova/plugin.compileCoffeescript.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at Object.stringify
   (/Users/noahgrumman/.meteor/packages/less/.2.5.1.196vlrt++os+web.browser+web.cordova/plugin.compileLessBatch.os/npm/ecmascript-runtime/node_modules/meteor-ecmascript-runtime/node_modules/core-js/modules/es6.symbol.js:127:21)
   at BrowserifyPlugin.buildErrorMessage (<runJavaScript-61>:177:70)
   at BrowserifyPlugin.compileOneFile (<runJavaScript-61>:142:42)
   at <runJavaScript-59>:555:47
   at <runJavaScript-59>:585:11
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:324:29
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:44:16
   at <runJavaScript-59>:591:9
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:182:20
   at replenish
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:317:21)
   at
   /Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:328:15
   at Object.async.forEachLimit.async.eachLimit
   (/Users/noahgrumman/.meteor/packages/cosmos_browserify/.0.9.3.tvgd37++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/caching-compiler/node_modules/async/lib/async.js:221:35)
   at BrowserifyPlugin.processFilesForTarget (<runJavaScript-59>:530:11)
```

**Comment by elidoran** [comment]
This seems to be a special problem because an error is occurring and then the `buildErrorMessage` is using `JSON.stringify()` which is erroring. So, it's not showing the original error because the error reporting function is failing. The error reporting is failing because the object has a circular reference. Yay. :-/

I will change it to use the [json-stringify-safe](https://github.com/isaacs/json-stringify-safe) module to avoid the problem.

**Comment by NoahGrumman** [comment]
Okay, thanks. I look forward to using it once this has been fixed.

**Comment by elidoran** [comment]
published in 0.9.4
